Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -79,88 +73,44 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
   }
 }
 
-- (void)loadWindow {
-  [super loadWindow];
-  NSURL *url = [SNTBlockMessage eventDetailURLForEvent:self.event customURL:self.customURL];
-
-  if (!url) {
-    [self.openEventButton removeFromSuperview];
-  } else if (self.customURL.length == 0) {
-    // Set the button text only if a per-rule custom URL is not used. If a
-    // custom URL is used, it is assumed that the `EventDetailText` config value
-    // does not apply and the default text will be used.
-    NSString *eventDetailText = [[SNTConfigurator configurator] eventDetailText];
-    if (eventDetailText) {
-      [self.openEventButton setTitle:eventDetailText];
-      // Require the button keyEquivalent set to be CMD + Return
-      [self.openEventButton setKeyEquivalent:@"\r"];  // Return Key
-      [self.openEventButton
-        setKeyEquivalentModifierMask:NSEventModifierFlagCommand];  // Command Key
-    }
-  }
-
-  NSString *dismissButtonText = [[SNTConfigurator configurator] dismissText];
-  if (dismissButtonText.length) {
-    [self.dismissEventButton setTitle:dismissButtonText];
-  }
-
-  if (!self.event.needsBundleHash) {
```

Comment:
Mind adding this if/else block as a comment in SNTBinaryMessageWindowView.swift? Just so we can remember what is should be doing when we go to implement.